torchvision/prototype/datasets/_builtin/__init__.py

```diff
@@ -8,10 +8,11 @@
 from .fer2013 import FER2013
 from .gtsrb import GTSRB
 from .imagenet import ImageNet
-from .mnist import MNIST, FashionMNIST, KMNIST, EMNIST, QMNIST
+from .mnist import EMNIST, FashionMNIST, KMNIST, MNIST, QMNIST
```

Comment:
Could you revert this? I'm guessing you are using `usort>=1`. We are currently bound to `usort==0.6.4` until #5106 is merged.